src/store/Company.js

```diff
@@ -15,7 +15,7 @@ const mutations = {
 const actions = {
     getData() {
         return axios({
-            url: "/companies"
+            url: "/companies?relationships=app,hasActivities"
```

Comment:
```suggestion
             url: "/companies?relationships=apps"
```

`hasActivities` should be a property on the company, not a relationship.